app/SurveyResponse.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class SurveyResponse extends Model
+{
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     *
+     * Uses questions created in Question Model
+     */
+    public function questions()
+    {
+        return $this->hasMany(Question::class);
+    }
+
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     * Get the user who submitted the survey response
+     */
+    public function user()
+    {
+        return $this->belongsTo(User::class);
+    }
```

Comment:
Is this for the user who took the survey or for the respondents?

Reply:
User in this context is the person conducting the survey (the enumerator). Every survey response that is submitted by an enumerator (user) should be tagged to that enumerator.

Reply:
I think this may not be necessary since a response belongs to a survey, we should pull the user who submitted from the survey itself. We should have the user who collects/submits the survey data as the user responsible for all the survey data otherwise we will have to track down whoever collected each response and that can get tedious.
In a real scenario, there's always one overall person responsible for a given survey. If we want to also note the people who participated in collecting the data, then we could have something like a survey_team table to hold that info.